docs/userGuide/deployingTheSite.md

```diff
@@ -49,6 +49,40 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 `markbind deploy` does not generate the static site from your source; it simply deploys the files that are already in the `_site` directory. You need to run `markbind build` first if you want to generate the site before deploying.
 </box>
 
+### Deploying to GitHub Pages via Travis CI
+**You can setup [<tooltip content="a platform for Continous Integration and Delivery (among other things)">Travis CI</tooltip>](https://www.travis-ci.org/) to automatically build and deploy your site on GitHub Pages every time your GitHub repo is updated.**
+
+Here are the steps to set up Travis CI:
+
+1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.org).
+1. Accept the autorisation for Travis CI when you are redirected to GitHub.
+1. Click the green _Activate_ button, and select the repository with the MarkBind site.
+1. [Generate a GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token) with **repo** permissions. Take note of the generated token - you will not be able to see it again once you navigate away from the page.
+1. [Add an environment variable in Travis CI](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings) named `GITHUB_TOKEN`, with the value set to the personal access token generated in the previous step. Ensure that _Display value in the build log_ is set to _Off_.
```

Comment:
Add this screenshot (or something similar) here?
![image](https://user-images.githubusercontent.com/1673303/52037608-8171f100-256a-11e9-9c00-f402cdcea17d.png)